docs/extensions/index.rst

```diff
@@ -0,0 +1,35 @@
+Sylius Extensions
+=================
+
+Sylius as a platform has a lot of space for various customizations and extensions. It aims to provide a simple schema for
+developing extensions(plugins). Anything you can imagine can be implemented and added to the Sylius framework as an extension.
+
+.. toctree::
+    :hidden:
+
+    creating-extension
+    list
+
+What are the extensions for?
+----------------------------
+
+The extensions (or Plugins like some call them) are Symfony Bundles that either modify or extend Sylius default behaviour,
+providing useful features that are built on top of the Sylius Core.
+
+Exemplary features may be: Social media buttons, newsletter, wishlists etc.
+
+Extension Development Flow
+--------------------------
+
+Since Sylius is an open-source platform, there is a certain flow in order for the extension to become officially adopted by the community.
+
+**1.** Develop the extension using :doc:`the official Extension Development guide </extensions/creating-extension>`.
+
+**2.** Send it to the project maintainers. It can be via email, Slack or even GitHub issues.
```

Comment:
Would it make sense to add links to these contact methods? Where to find more info about the Slack channel oder sth. like that?